tds/build.gradle

```diff
@@ -85,10 +85,10 @@ dependencies {
   'org.json:json'
 
   // JSR 303 with Hibernate Validator, which is dragging in jboss logging
-  compile 'javax.validation:validation-api'
-  runtime 'org.hibernate.validator:hibernate-validator'
+  compile 'jakarta.validation:jakarta.validation-api'
+  compile 'org.hibernate.validator:hibernate-validator'
   runtime 'javax.el:javax.el-api'
-  runtime 'org.glassfish:javax.el'
+  compile 'org.glassfish:jakarta.el'
```

Comment:
should this be `runtime`?

Reply:
No, unfortunately, it won't find the lib unless it is specified as compile.  :-|